vuepress/docs/next/docs/getting-started/ent-diag.md

```diff
@@ -0,0 +1,20 @@
+---
+sidebarDepth: 1
+---
+
+# Diagnostics and Troubleshooting
+
+Diagnostic information is critical to analysis and troubleshooting, and the Entando CLI (ent) is a useful tool for debugging issues with an Entando instance. The [Getting Started](../../docs/getting-started/README.md) guide includes steps to install ent in a Multipass VM. The following commands can be run from this VM for insight into an Entando Application.
```

Comment:
I think we can remove the 2nd and 3rd sentences. These commands are useful against any Entando instance, not just a Multipass VM.